packages/node_modules/@ciscospark/react-component-file-staging-area/src/index.js

```diff
@@ -29,6 +29,7 @@ function FileStagingArea({
 
   const fileChips = [];
 
+  // eslint-disable-next-line react/prop-types
```

Comment:
Are the prop types wrong on this? Code seems to indicate that `files` is an array.

Reply:
For some reason it's indicating that `forEach` is not a builtin, but a prop that needs to be validated.
Not entirely sure what the solution here was besides just ignoring it

Reply:
Gotcha, yea let's ignore it. Looking at the tests it is an immutable map that is expected.